05-onchain.md

```diff
@@ -2,15 +2,29 @@
 
 ## Abstract
```

Comment:
OK, I used wdiff to make sure there were no accidental textual changes here; the only change was a FIXME to a [FIXME].

Just a heads-up that @shannona doesn't need to review this patch :)